assessmentModel/src/jvmMain/kotlin/serialization/TextFormatter.kt

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.assessmentmodel.serialization
+
+import kotlinx.serialization.json.*
+import org.sagebionetworks.assessmentmodel.survey.FormattedValue
+import org.sagebionetworks.assessmentmodel.survey.NumberType
+import org.sagebionetworks.assessmentmodel.survey.TextValidator
+import java.text.NumberFormat
+import java.text.ParseException
+
+//TODO: syoung 02/13/2020 implment the other styles of formatting. (Spellout, Scientific, Ordinal)
```

Comment:
Is this a copy/paste or a rename fail?

Reply:
This is a copy past of the Android implementation. Currently there isn't an easy way to share implementations between Android and JVM targets.